service/collector/collector.go

```diff
@@ -115,6 +130,7 @@ func (c *Collector) Collect(ch chan<- prometheus.Metric) {
 	collectFuncs := []func(chan<- prometheus.Metric, []awsutil.Clients){
 		c.collectClusterInfo,
 		c.collectAccountsVPCs,
+		c.collectAccountsELBs,
```

Comment:
Registering the new collector.

Reply:
Having this alphabetical would be nice 🐱

Reply:
FMI, can you suggest an edit for flipping two lines or can you only suggest an edit for a single line only?

Reply:
Done.